releasenotes/notes/release-1.0-758fc917e40d7eb3.yaml

```diff
@@ -15,69 +15,227 @@ prelude: >
-    :ref:`Removed <remove-span-init-tracer>`,``ddtrace.Span.__init__(tracer=)``
-    :ref:`Removed <remove-span-meta>`,``ddtrace.Span.meta``
-    :ref:`Removed <remove-span-metrics>`,``ddtrace.Span.metrics``
-    :ref:`Removed <remove-span-set-meta>`,``ddtrace.Span.set_meta``
-    :ref:`Removed <remove-span-set-metas>`,``ddtrace.Span.set_metas``
-    :ref:`Removed <remove-span-pprint>`,``ddtrace.Span.pprint``
-    :ref:`Removed <remove-pin-app>`,``ddtrace.Pin.app``
-    :ref:`Removed <remove-pin-apptype>`,``ddtrace.Pin.app_type``
-    :ref:`Removed <remove-clone-context>`,``ddtrace.context.Context.clone``
-    :ref:`Removed <remove-ddtrace-compat>`,``ddtrace.compat``
-    :ref:`Removed <remove-contrib-util>`,``ddtrace.contrib.util``
-    :ref:`Removed <remove-ddtrace-encoding>`,``ddtrace.encoding``
-    :ref:`Removed <remove-span-types-enum>`,``ddtrace.ext.SpanTypes``
-    :ref:`Removed <remove-ext-errors>`,``ddtrace.ext.errors``
-    :ref:`Removed <remove-ext-priority>`,``ddtrace.ext.priority``
-    :ref:`Removed <remove-ext-system>`,``ddtrace.ext.system``
-    :ref:`Removed <remove-helpers>`,``ddtrace.helpers.get_correlation_ids``
-    :ref:`Removed <remove-http>`,``ddtrace.http``
-    :ref:`Removed <remove-ddtrace-monkey>`,``ddtrace.monkey``
-    :ref:`Removed <remove-ddtrace-propagation-utils>`,``ddtrace.propagation.utils``
-    :ref:`Removed <remove-default-sampler>`,``ddtrace.Sampler.default_sampler``
-    :ref:`Removed <remove-config-httpserver>`,``ddtrace.settings.Config.HTTPServerConfig``
-    :ref:`Removed <remove-ddtrace-util>`,``ddtrace.util``
-    :ref:`Removed <remove-ddtrace-utils>`,``ddtrace.utils``
-    :ref:`Removed <remove-cassandra-traced>`,``ddtrace.contrib.cassandra.get_traced_cassandra``
-    :ref:`Removed <remove-celery-patch-task>`,``ddtrace.contrib.celery.patch_task``
-    :ref:`Removed <remove-celery-unpatch-task>`,``ddtrace.contrib.celery.unpatch_task``
-    :ref:`Removed <remove-mongoengine-traced>`,``ddtrace.contrib.mongoengine.trace_mongoengine``
-    :ref:`Removed <remove-mysql-legacy>`,``ddtrace.contrib.mysql.get_traced_mysql_connection``
-    :ref:`Removed <remove-psycopg-legacy>`,``ddtrace.contrib.psycopg.connection_factory``
-    :ref:`Removed <remove-requests-legacy-distributed>`,``ddtrace.contrib.requests.legacy``
-    :ref:`Removed <remove-sqlite3-legacy>`,``ddtrace.contrib.sqlite3.connection_factory``
+
+
+  Upgrading summary
+
+  ~~~~~~~~~~~~~~~~~
+
+  Removed legacy environment variables
+
+  ************************************
+
+  These environment variables have been removed. In all cases the same
+  functionality is provided by other environment variables and replacements are
+  provided as recommended actions for upgrading.
+
+
+  .. list-table::
+    :header-rows: 1
+    :widths: 45, 45, 10
+
+    * - Variable
+      - Replacement
+      - Note
+    * - ``DATADOG_`` prefix
+      - ``DD_`` prefix
+      - :ref:`📝<remove-datadog-envs>`
```

Comment:
Preferably we should use `|:memo:|` provided by `sphinxemoji` but the replacement does not seem to work within a `:ref:` context.

By the looks of it, the emoji short code should be replaced before the cross-reference link is transformed:

https://docutils.sourceforge.io/docs/ref/transforms.html#transform-priority-range-categories

https://github.com/sphinx-contrib/emojicodes/blob/c536aee39972a4dfa6d2111120f50411b8fd3bee/sphinxemoji/sphinxemoji.py#L49

https://github.com/sphinx-doc/sphinx/blob/4221d1a5163d023d80a34bd4a12e9c2a2c648ae3/sphinx/transforms/references.py#L30